frontend/src/components/dashboard/home/VisitsDensityWidget.vue

```diff
@@ -0,0 +1,216 @@
+                    },
+                    legend: {
+                        show: false
+                    },
+                    plotOptions: {
+                        heatmap: {
+                            radius: 6,
+                        }
+                    },
+                    states: {
+                        hover: {
+                            filter: {
+                                type: 'none',
+                            }
+                        },
+                        active: {
+                            filter: {
+                                type: 'none',
+                            }
+                        },
+                    },
+                    tooltip: {
+                        enabled: true,
+                        custom: ({series, seriesIndex, dataPointIndex}) => {
+                            return `<span style="padding: 10px">${series[seriesIndex][dataPointIndex]}</span>`;
+                        }
+                    },
+                    xaxis: {
+                        labels: {
+                            show: true
+                        },
+                        axisBorder: {
+                            show: false,
+                        },
+                        axisTicks: {
+                            show: false,
+                        },
+                        tooltip: {
+                            enabled: false
+                        }
+                    },
+                    yaxis: {
+                        opposite: true,
+                        reversed: true,
+                        labels: {
+                            align: 'right'
+                        }
+                    }
+                }
+            };
+        },
+        computed: {
+            ...mapGetters('visits_density_widget', {
+                getSelectedPeriod: GET_SELECTED_PERIOD,
+                getVisitsData: GET_VISITS_DATA,
+                isFetching: IS_FETCHING
+            }),
+        },
+        created() {
+            this.fetchWidgetData();
+            this.drawHeatmap();
+        },
+        methods: {
+            ...mapActions('visits_density_widget', {
+                changeSelectedPeriod: CHANGE_SELECTED_PERIOD,
+                fetchWidgetData: FETCH_WIDGET_DATA
+            }),
+            drawHeatmap () {
+                if (_.isEmpty(this.getVisitsData)) {
+                    this.chartOptions.xaxis.labels.show = false;
+                    this.chartOptions.yaxis.labels.show = false;
+                    return [];
+                }
+
+                let series = [];
+
+                for (let hour = 0; hour < 24; hour++) {
+                    let row = {};
+                    row.name = this.hours[hour] || '';
+                    row.data = [];
+
+                    for (let day = 0; day < 7; day++) {
+                        let visitsData = (_.find(this.getVisitsData, { 'hour': hour, 'day': day }));
```

Comment:
It looks like items is an array, array has methods like `find`, so you probably no need to use lodash here